src/command/graph/check.rs

```diff
@@ -12,6 +12,7 @@ use crate::utils::parsers::{
     parse_graph_ref, parse_query_count_threshold, parse_query_percentage_threshold,
     parse_schema_source, parse_validation_period, GraphRef, SchemaSource, ValidationPeriod,
 };
+use crate::utils::table;
```

Comment:
would it make sense to re-export the cell and row macros from this util as well to keep the `use` declaration the same?

Reply:
done!